app/schemas/user.js

```diff
@@ -41,7 +41,31 @@ const signUp = {
   }
 };
 
+const listAllUsers = {
+  page: {
+    in: ['query'],
+    exists: {
```

Comment:
:thinking: This must be optional. If the request doesn't have a page you must return page 1.

Reply:
Solved!